packages/core/src/interactions/packet/index.spec.ts

```diff
@@ -25,8 +25,9 @@ import assert from 'assert'
 import { AcknowledgementChallenge, Packet, Acknowledgement } from '../../messages'
 import { PacketForwardInteraction } from './forward'
 import { initializeCommitment } from '@hoprnet/hopr-core-ethereum'
+import { SECRET_LENGTH } from '@hoprnet/hopr-utils/lib/crypto/packet/constants'
```

Comment:
Directly importing built Typescript files is dangerous and might not work in `hoprd`. Why don't export PacketConstants in `utils` and then import it from `@hoprnet/hopr-utils`?

Reply:
Packet Constants  have unfortunately naming clashes with other stuff. It would require more refactoring on multiple places.
Previously this unit test had just `const SECRET_LENGTH = 32` defined locally and didn't have an import, so I'll put it back for the time being.